c2corg_ui/externs/appx.js

```diff
@@ -58,7 +58,7 @@ appx.SearchDocumentLocale;
 
 /**
  * @typedef {{
- *  msg: string,
+ *  msg: (string|Object),
```

Comment:
Yes, I also expected @gberaudo would say something here :p 
That would make sense indeed, especially since the alerts service expects this is a ``response`` object with a ``response['data']['errors']`` property. But are the API responses always formatted the same way? I'm also afraid that the type is a bit complicated.